py/setup.cfg

```diff
@@ -10,6 +10,8 @@ url = https://github.com/torsteingrindvik/serial-keel
 classifiers =
     Programming Language :: Python :: 3
     Operating System :: OS Independent
+    License :: OSI Approved :: MIT License
+    License :: OSI Approved :: Apache Software License
 [options]
 packages = find:
 python_requires = >=3.6
```

Comment:
Are we sure about 3.6 working?
I think I've used 3.10 locally, and I can't remember/don't know if I've used anything that requires >3.6.

Reply:
I've used 3.10 locally as well, updated this to match.